config/locales/en.yml

```diff
@@ -241,3 +241,218 @@ en:
+          href: "https://www.gov.uk/government/news/government-agrees-measures-with-energy-industry-to-support-vulnerable-people-through-covid-19"
+        - text: "Get information on what do if you cannot pay your rent or mortgage from Shelter"
+          href: "https://england.shelter.org.uk/housing_advice/coronavirus#Rent_payment_problems"
+        - text: "Get information on what do if you cannot pay your rent, mortgage or bills from Citizens Advice"
+          href: "https://www.citizensadvice.org.uk/debt-and-money/if-you-cant-pay-your-bills-because-of-coronavirus/"
+        - text: "Get help from Housing Rights if you live in Northern Ireland"
+          href: "https://www.housingrights.org.uk/"
+        - text: "Get information if you're finding it hard to pay rent and you live in Wales"
+          href: "https://gov.wales/coronavirus-covid-19-guidance-for-tenants-in-the-private-rented-sector-html"
+        - text: "Get coronavirus housing information from Shelter Wales"
+          href: "https://sheltercymru.org.uk/get-advice/coronavirus/"
+        - text: "Get information about your rights if you have a private landlord in Scotland"
+          href: "https://www.mygov.scot/private-rental-rights/"
+        - text: "Get coronavirus housing information from Shelter Scotland"
+          href: "https://scotland.shelter.org.uk/get_advice/scottish_housing_advice_coronavirus_COVID_19"
+        - text: "Get information about your rights if you have a social landlord in Scotland"
+          href: "https://www.mygov.scot/social-rental-rights/"
+        - text: "Get coronavirus housing information from Shelter Scotland"
+          href: "https://scotland.shelter.org.uk/get_advice/scottish_housing_advice_coronavirus_COVID_19#Paying_rent"
+    afford_food:
+      show_options:
+        - "Yes"
+        - "Not sure"
+      links:
+        - text: "Find out if you’re eligible for Universal Credit"
+          href: "https://www.gov.uk/universal-credit/eligibility"
+        - text: "If you are at least 10 weeks pregnant or have a child under 4, find out if you can apply for Healthy Start vouchers"
+          href: "https://www.healthystart.nhs.uk/healthy-start-vouchers/do-i-qualify/"
+        - text: "Find out if you're eligible for the Scottish Welfare Fund if you live in Scotland"
+          href: "https://www.mygov.scot/scottish-welfare-fund/"
+        - text: "Find out if you can get a grant if you're a parent or looking after a child in Scotland"
+          href: "https://www.mygov.scot/best-start-grant-best-start-foods/"
+        - text: "Get information about getting free school meals during coronavirus if you live in Scotland"
+          href: "https://www.mygov.scot/school-meals/"
+        - text: 'If you are in Northern Ireland call the coronavirus Community Helpline on <a href="tel:0808 802 0020" class="govuk-link">0808 802 0020</a>, email covid19@adviceni.net or text ‘ACTION’ to 81025'
```

Comment:
`tel:` links need to include the country code:

```suggestion
        - text: 'If you are in Northern Ireland call the coronavirus Community Helpline on <a href="tel:+448088020020" class="govuk-link">0808 802 0020</a>, email covid19@adviceni.net or text ‘ACTION’ to 81025'
```

Reply:
great spot

Reply:
good spot @injms! 🙌